libs/components/flyout/src/lib/modules/flyout/flyout.component.scss

```diff
@@ -101,22 +201,167 @@ $header-height: 50px;
 }
 
 .sky-flyout-header {
-  @include mixins.sky-border(dark, bottom);
+  border-bottom: var(
+    --sky-override-flyout-header-border-bottom,
+    var(--sky-border-width-divider) var(--sky-border-style-divider)
+      var(--sky-color-border-divider)
+  );
   width: inherit;
-  background: $sky-background-color-neutral-light;
-  height: $header-height;
+  background: var(
+    --sky-override-flyout-header-background,
+    var(--sky-color-background-container-base)
+  );
+  padding: var(
+    --sky-override-flyout-header-padding,
+    var(--sky-space-inset-letterbox-2_3-top-m)
+      var(--sky-space-inset-letterbox-2_3-right-m)
+      var(--sky-space-inset-letterbox-2_3-bottom-m)
+      var(--sky-space-inset-letterbox-2_3-left-m)
+  );
+  height: var(--sky-override-flyout-header-height);
```

Comment:
Chatted offline - but we are removing a specified height here and instead using `position: sticky` to make the header just conform to its content and still allow for flyout content scrolling. 

A relevant change in the `sky-flyout-content` selector below is the removal of `margin-top` and the height now being set to `100%` instead of `100% - 50px`